deploy/clowdapp.yaml

```diff
@@ -23,6 +23,12 @@ objects:
   stringData:
     client-id: ${ID_CLOUD_CONNECTOR}
     client-psk: ${KEY_CLOUD_CONNECTOR}
+- apiVersion: v1
```

Comment:
as this is only used by connect, this should go the the connect template

Reply:
All done! :+1: